SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex

```diff
@@ -969,3 +972,102 @@ \subsection{CMakeList.txt}
 set(ITKVideoBridgeOpenCV_SYSTEM_INCLUDE_DIRS ${OpenCV_INCLUDE_DIRS})
 set(ITKVideoBridgeOpenCV_SYSTEM_LIBRARY_DIRS ${OpenCV_LIB_DIR})
 \end{minted}
+
+\section{Contributing with a Remote Module}
+\label{sec:ContributingRemoteModules}
+
+For most ITK community members, the modularization of the toolkit is relatively
+transparent. The default configuration includes all the (default) modules into
+the ITK library, which is used to build their own ITK applications.
+
+For ITK developers and code contributors, the modular structure imposes rules
+for organizing the source code, building the library and contributing to the
+ITK source code.
+
+A Module may be distributed outside the main ITK repository, but it may be
+built into ITK as a Remote Module once downloaded into the local copy of
```

Comment:
built into ITK as a Remote Module once downloaded into the local copy of ITK source tree -> made available in the ITK repository as a Remote Module.

Reply:
Done.